Common/src/main/java/com/blamejared/crafttweaker/natives/villager/ExpandMerchantOffer.java

```diff
@@ -0,0 +1,213 @@
+        constructors = {
+                @NativeConstructor({
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "baseCostA"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "result"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "uses"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "maxUses"),
+                }),
+                @NativeConstructor({
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "baseCostA"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "costB"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "result"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "maxUses"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "xp"),
+                        @NativeConstructor.ConstructorParameter(type = Float.class, name = "priceMultiplier"),
+                }),
+                @NativeConstructor({
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "baseCostA"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "costB"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "result"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "uses"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "maxUses"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "xp"),
+                        @NativeConstructor.ConstructorParameter(type = Float.class, name = "priceMultiplier"),
+                }),
+                @NativeConstructor({
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "baseCostA"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "costB"),
+                        @NativeConstructor.ConstructorParameter(type = ItemStack.class, name = "result"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "uses"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "maxUses"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "xp"),
+                        @NativeConstructor.ConstructorParameter(type = Float.class, name = "priceMultiplier"),
+                        @NativeConstructor.ConstructorParameter(type = Integer.class, name = "demand"),
+                })
+        })
```

Comment:
Hm... these vs a builder or so 🤔 
But that may be easier to find since it would be on the same page then...